hw3/hw3.py

```diff
@@ -1,101 +1,113 @@
-import os
 import sys
+from pyspark import StorageLevel
+from pyspark import keyword_only
+from pyspark.sql import SparkSession
+from pyspark.ml import Transformer
+
+
+# Create a spark session
+spark = SparkSession.builder \
+    .config("spark.jars",
+            "/Users/zack/Documents/SDSU/Fall 2022/BDA 602/src/bda602/hw3/mysql-connector-java-5.1.46/mysql-connector-java-5.1.46.jar") \
+    .master("local") \
+    .appName("HW3").getOrCreate()
+
+# Define Transformer Class
+
+class RollingAverageTransform(Transformer):
```

Comment:
Great Job on creating an object that does the calculations for you. It makes the code look very clean and it will be a lot easier to implement this code into another script if necessary.